dashboards-observability/public/components/synthetics/home_view.tsx

```diff
@@ -138,75 +289,21 @@ export const SyntheticHomeTab = (props: LogHistoryProps) => {
           tempVal.push(res[type][field])
         };
         result(tempVal)
-        console.log(tempVal)
       })
       .catch((error: Error) => {
-        // setIsError(error.stack);
         result([])
         console.error(error);
       })
-      .finally(() => {
-        // setIsLoading(false);
-      });
   };
 
-  // columns for the History table
-  const columns = [
-    {
-      field: "monitorName",
-      name: "Monitor Name",
-      render: (monitor: {} | null | undefined) => (
-        <EuiLink href={`#${monitor}`} target="_blank">
-          {monitor}
-        </EuiLink>
-      ),
-      mobileOptions: {
-        show: false
-      },
-    },
-    {
-      field: "status",
-      name: "Status",
-    },
-    {
-      field: "URL",
-      name: "URL",
-      render: (url: {} | null | undefined) => (
-        <EuiLink href={`${url}`} target="_blank">
-          {url}
-        </EuiLink>
-      ),
-      mobileOptions: {
-        show: false
-      },
-    },
-    {
-      field: "response.status",
-      name: "Status Code",
-    },
-    {
-      field: "startTime",
-      name: "Latest Time Polled",
-      type: "date",
-      render: (date: Date) => {
-        var datet: Date = new Date(date)
-        // needed to convert date into local time, as the index stores times in UTC and this page
-        // incorrectly reads it as local time (while being UTC) so the time will be off by the timezoneoffset
-        var utcdate: Date = new Date(datet.getTime() - (datet.getTimezoneOffset() * 60 * 1000))
-        return utcdate.toString()
-      },
-    },
-  ];
-
   // will pull from index logs every 5 seconds to update at that interval
   useEffect(() => {
+    updateQueryComponents();
     const interval = setInterval(() => {
```

Comment:
Maybe put the setInterval within its own useEffect with an empty dependency array.

Reply:
Added.